File: 백준/Bronze/32905. RACI/RACI.js
const fs = require("fs");

function getNextLine(input) {
    let inputIdx = 0;
    return function () {
        return input[inputIdx++].split(/\s+/);
    };
}

const input = fs.readFileSync(0, "utf8").trim().split("\n");

const nextLine = getNextLine(input);

const [n, m] = nextLine().map(Number);

let isValid = true;

for (let i = 0; i < n; i++) {
    let accountableCount = 0;
    const task = nextLine();
    for (const person of task) {
        if (person === "A") accountableCount++;
    }

    if (accountableCount !== 1) {
        isValid = false;
        break;
    }
}

console.log(isValid ? "Yes" : "No");

File: 백준/Bronze/32905. RACI/RACI.py
import sys

input = sys.stdin.readline

n, m = map(int, input().split())

is_valid = True

for _ in range(n):
    task = input().split()
    accountable_count = task.count("A")

    if accountable_count != 1:
        is_valid = False
        break

print("Yes" if is_valid else "No")
